pkg/autoscaler/autoscaler.go

```diff
@@ -204,25 +208,47 @@ func remaining(t time.Time) time.Duration {
 }
 
 var _ = Describe("[Feature:Machines] Autoscaler should", func() {
+
+	var workloadMemRequest resource.Quantity
+	var client runtimeclient.Client
+	var err error
+
+	BeforeEach(func() {
+		client, err = framework.LoadClient()
+		Expect(err).NotTo(HaveOccurred())
+
+		workerNodes, err := framework.GetWorkerNodes(client)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(len(workerNodes)).To(BeNumerically(">=", 1))
+
+		memCapacity := workerNodes[0].Status.Capacity[corev1.ResourceMemory]
+		Expect(memCapacity).ShouldNot(BeNil())
+		Expect(memCapacity.String()).ShouldNot(BeEmpty())
+		glog.Infof("Memory capacity of worker node %q is %s", workerNodes[0].Name, memCapacity.String())
+
+		bytes, ok := memCapacity.AsInt64()
+		Expect(ok).Should(BeTrue())
+
+		// 70% - enough that the existing and new nodes will
+		// be used, not enough to have more than 1 pod per
+		// node.
+		workloadMemRequest = resource.MustParse(fmt.Sprintf("%v", 0.7*float32(bytes)))
+	})
+
 	cascadeDelete := metav1.DeletePropagationForeground
+	deleteObject := func(name string, obj runtime.Object) error {
+		glog.Infof("[cleanup] %q (%T)", name, obj)
+		return client.Delete(context.TODO(), obj, &runtimeclient.DeleteOptions{
```

Comment:
Can we either replace TODO with Background or declare `var ctx = client.Background()` on line 215 and use that everywhere? Nothing should be modifying the `ctx` variable so I think this is safe and this is a pattern I've seen used in a lot of test suites within kubernetes-sigs etc